src/workerd/tests/bench-global-scope.c++

```diff
@@ -41,5 +41,39 @@ BENCHMARK_F(GlobalScopeBenchmark, request)(benchmark::State& state) {
   }
 }
 
+struct QueueMicrotaskBenchmark: public benchmark::Fixture {
+  virtual ~QueueMicrotaskBenchmark() noexcept(true) {}
+
+  void SetUp(benchmark::State& state) noexcept(true) override {
+    TestFixture::SetupParams params = {.mainModuleSource = R"(
+        export default {
+          async fetch(request) {
+            queueMicrotask(() => {
+              throw new Error('boom');
+            });
+            return new Response("OK");
+          },
+        };
+      )"_kj};
+    fixture = kj::heap<TestFixture>(kj::mv(params));
+  }
+
+  void TearDown(benchmark::State& state) noexcept(true) override {
+    fixture = nullptr;
+  }
+
+  kj::Own<TestFixture> fixture;
+};
+
+BENCHMARK_F(QueueMicrotaskBenchmark, request)(benchmark::State& state) {
+  for (auto _: state) {
+    for (size_t i = 0; i < 1000; ++i) {
```

Comment:
you don't need this for loop. the benchmark will run 1second regardless of this for loop.

Reply:
I think you misunderstood. 


We're calling google benchmarks with --benchmark_min_time=1s. regardless of the size of this input it will run exactly 1 seconds. no less... so this for loop of size 1000 will only change how many times the for state loop is executed, and rather than showing you the duration of this single line, google benchmark will show you the duration of 1000 requests. 

Please take a look at https://github.com/cloudflare/workerd/blob/main/.github/workflows/codspeed.yml#L56

Reply:
No, I completely understand what I'm doing here. I *intentionally* modeled the new bench after the existing bench in the same file because I'm not only comparing changes to `queueMicrotask`, I'm comparing the impact of adding `queueMicrotask` with a throw to the existing benchmark.